go.mod

```diff
@@ -1,3 +1,3 @@
 module github.com/microsoft/go-crypto-winnative
 
-go 1.16
+go 1.17
```

Comment:
`unsafe.Slice` is available since go1.17. We don't use this backend on go1.16, so this change is harmless.